platform/security/sts/security-sts-pkivalidator/src/test/java/org/codice/ddf/security/validator/pki/TestPKITokenValidator.java

```diff
@@ -213,6 +213,13 @@ public void testValidateToken() {
         assertEquals(ReceivedToken.STATE.VALID,
                 tokenValidatorResponse.getToken()
                         .getState());
+
+        assertEquals("US",
```

Comment:
According to our Unit Testing guidelines, we should favor `assertThat`over `assertEquals` and the ordering should be `actual, expected`